app/views/rails_admin/main/_form_nested_many.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="controls col-sm-10" data-nestedmany="true">
   <div class="btn-group">
-    <a class="<%= (field.active? ? 'active' : '') %> btn btn-info toggler" data-target="<%= form.jquery_namespace(field) %> > .tab-content, <%= form.jquery_namespace(field) %> > .controls > .nav" data-toggle="button">
+    <a class="<%= (field.active? ? 'active' : 'active') %> btn btn-info toggler" data-bs-target="<%= form.jquery_namespace(field) %> .collapse" data-bs-toggle="collapse" role="button">
```

Comment:
This `active` is my mistake - I forgot to remove it after testing behavior when active.